### ISSUE TITLE
database updates with special characters

### ISSUE THREAD

**Issue by Thomas-O** [jcorporation/myMPD#1334]
myMPD version: 16.1.0

It's impossible to add files containing numbers or special characters in the file names.

Server platform:
 - OS: DietPi
 - OS version v9.6
 - Platform: armv7l
 - MPD version: 0.23.5

**Comment by jcorporation** [comment]
I do not understand. Please describe your issue in more detail. Screenshot or logs?

**Comment by jcorporation** [comment]
No response

**Comment by Thomas-O** [comment]
> No response

Sorry for the incredibly slow reply. My PC died on the day I posted the above. I left the room to make a drink and the CPU fan failed. Then shortly after the CPU went the same way. It took me 2 weeks to get a new PC, new operating system and all the rest. And when I got up and running I started looking in to the issue. Going down one rabbit hole after another. Until I eventually twigged that the problem was with the operating system I was using to host myMPD.

I was using a converted version of DietPi. Converted from Armbian. There's either a problem with the Armbian image or the conversion. Which only became obvious when I bought a new RPI and installed DietPi + myMPD and found that everything is OK. No problems with special characters. Absolutely fine.

Sorry for being a bother.
